src/main/resources/hudson/plugins/gradle/injection/InjectionConfig/help-enforceUrl.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Whether to enforce the Gradle Enterprise Server URL configured in this connection over a URL configured in the project's build.
```

Comment:
It must be phrased differently I think. There is no such thing as a connection. Connection is a TC-specific thing.